common/thorhelper/thorxmlread.hpp

```diff
@@ -196,7 +196,11 @@ extern thorhelper_decl ICsvToRawTransformer * createCsvRawTransformer(ICsvToRowT
 #ifdef _USE_ICU
 #include "unicode/utf.h"
 #else
+#ifdef _WIN32
```

Comment:
@richardkchapman these uchar changes may require some more work to enable it to build properly with icu.  I think out external libraries have defined it one way, but the default vcpkg icu has it defined another.  I will revisit in a later PR.

Does this cause problems building the existing client tools on windows?